### PR TITLE
added notebook for dXCT 22

### PR DESCRIPTION
**Pull request by paskino**
I put it in misc, maybe we need a better directory?

### CODE REVIEW

File: misc/Walnut2d.ipynb [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+   "source": [
+    "recon = recon3d.get_slice(vertical='centre')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ae67afb-0843-43e9-99de-d58b4c6c0df4",
+   "metadata": {},
+   "source": [
+    "Reduce the amount of projection to 10% by selecting a projection every 10."
```
Comment (lauramurgatroyd) [inline misc/Walnut2d.ipynb:160]:
```suggestion
    "Reduce the amount of projections to 10% by selecting 1 projection in every 10."
```

File: misc/Walnut2d.ipynb [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b62f206b-9328-4f4c-be5e-3644871fcd90",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "A = ProjectionOperator(ig,data_reduced.geometry)"
```
Comment (lauramurgatroyd) [inline misc/Walnut2d.ipynb:233]: Here I get an error that ig is not defined

**Review by lauramurgatroyd — CHANGES_REQUESTED** [review]

**Comment by lauramurgatroyd** [comment]
Not sure if it's just me but i reloaded a couple of times and still having a problem with the equations not formatting correctly:
image

**Comment by paskino** [comment]
Maybe this can become our first tutorial

**Comment by jakobsj** [comment]
@paskino Is this one still relevant? If it was used at dXCT we should get it in to have a record of it.

**Comment by lauramurgatroyd** [comment]
If I remember correctly, I don't think it was used in the end?

**Pull request closed without merging**